test/data/gvcf_test_cases/format_fields_default_missing.yml

```diff
@@ -0,0 +1,91 @@
+readme: |
+  This is a gvcf testing of the service workflow (discover_alleles, unified_sites, genotype_sites), with inclusion of FORMAT field liftover.
+  This test explicitly tests the behavior of default_type: missing, as applied to the SB field. The output includes a site that has no defined
+  SB field in sample A, and we expect the output SB field to be filled by "missing" value (ie a period '.').
+
+input:
+  header: |-
+      ##fileformat=VCFv4.1
+      ##FILTER=<ID=PASS,Description="All filters passed">
+      ##ALT=<ID=NON_REF,Description="Represents any possible alternative allele at this location">
+      ##FILTER=<ID=LowQual,Description="Low quality">
+      ##FORMAT=<ID=AD,Number=.,Type=Integer,Description="Allelic depths for the ref and alt alleles in the order listed">
+      ##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth (reads with MQ=255 or with bad mates are filtered)">
+      ##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
+      ##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+      ##FORMAT=<ID=MIN_DP,Number=1,Type=Integer,Description="Minimum DP observed within the GVCF block">
+      ##FORMAT=<ID=PGT,Number=1,Type=String,Description="Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another">
+      ##FORMAT=<ID=PID,Number=1,Type=String,Description="Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group">
+      ##FORMAT=<ID=PL,Number=G,Type=Integer,Description="Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification">
+      ##FORMAT=<ID=SB,Number=4,Type=Integer,Description="Per-sample component statistics which comprise the Fisher's Exact Test to detect strand">
+      ##FORMAT=<ID=SBF,Number=4,Type=Float,Description="(Fake field, SB as float) Per-sample component statistics which comprise the Fisher's Exact Test to detect strand">
```

Comment:
This is just a fake field to test float `set_to_missing` actually works... and I don't get a `23992.0` or the like.